crates/holochain_types/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- BREAKING CHANGE - Added zome name to the signal emitted when using `emit_signal`.
```

Comment:
It might be worth it to add that this changed from a tuple to named properties as well, as that’s another aspect of the breaking change